apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/Editor/Simple/TextEvaluationEditor/index.tsx

```diff
@@ -121,54 +67,47 @@ export default function TextEvaluationEditor({
   }
 
   return (
-    <div className='flex flex-col gap-y-2 h-full'>
-      <div className='flex flex-row items-center justify-between'>
-        <Text.H4M>{evaluation.name}</Text.H4M>
-        <Button fancy form='simple-text-evaluation-editor' type='submit'>
-          Save changes
-        </Button>
+    <EvaluationEditorLayout
+      evaluationId={evaluation.id}
+      name={evaluation.name}
+      formId='simple-text-evaluation-editor'
+      onSubmit={onSubmit}
+    >
+      <FormField>
+        <ProviderModelSelector
+          modelDisabled={!modelOptions.length || !selectedProvider}
+          modelOptions={modelOptions}
+          onModelChange={onModelChange}
+          onProviderChange={onProviderChange}
+          providerDisabled={!providerOptions.length}
+          providerOptions={providerOptions}
+          selectedModel={selectedModel}
+          selectedProvider={selectedProvider}
+        />
+      </FormField>
+      <FormField label='Evaluation objective'>
+        <Input
```

Comment:
Remember about `FormField` being unnecessary next time, please. Let's don't put bad examples to follow in other places